source/developers/references/internal_data_formats/tracking_log_event_list.rst

```diff
@@ -0,0 +1,504 @@
+.. _tracking_log_event_list:
+
+######################################
+Alphabetical Tracking Log Event List
+######################################
+
+
+:ref:`ABC` - :ref:`DEF` - :ref:`GHI` - :ref:`JKL` - :ref:`MNO`
+- :ref:`PQR` - :ref:`ST` - :ref:`UVWXYZ`
+
+.. _ABC:
+
+*******
```

Comment:
NIT: I don't think the overlines here serve much of a purpose but fine to leave them if you like them stylistically.